extension/content.js

```diff
@@ -263,6 +263,25 @@ function addPatchDiffLinks() {
 	`);
 }
 
+// Prompt user to confirm erasing a comment with the Cancel button
+$(document).on('click', function (event) {
+	// Check event.target instead of using a delegate, because Sprint doesn't support them
```

Comment:
Well, you can probably also remove `dimensions`, `deferred`, `event/alias` and `offset`, but I doubt it will save much.